lib/components/Keyboard/KeyboardInput/KeyboardAccessoryView.tsx

```diff
@@ -78,10 +67,10 @@ export type KeyboardAccessoryViewProps = kbTrackingViewProps & {
  * @gif: https://github.com/wix/react-native-ui-lib/blob/master/demo/showcase/KeyboardAccessoryView/KeyboardAccessoryView.gif?raw=true
  */
 class KeyboardAccessoryView extends Component<KeyboardAccessoryViewProps> {
-  static iosScrollBehaviors = IOS_SCROLL_BEHAVIORS;
+  // @ts-expect-error
```

Comment:
You can fix this by changing the export in `KeyboardTrackingView` to:
```
export default KeyboardTrackingView as (typeof KeyboardTrackingView & {scrollBehaviors: typeof SCROLL_BEHAVIORS});
```

Reply:
Thanks